pkg/virt-launcher/virtblocks/virtblocks.go

```diff
@@ -0,0 +1,45 @@
+package virtwrap
```

Comment:
@jnsnow @jnsnow this file contains most of the things of what we will need one or the other form virtblocks, functionality-wise. What is missing here is "monitoring" the process and sending events. Still extracting this part.

Reply:
@andreabolognani FYI

Reply:
I've been keeping up with the conversation, though I've just skimmed the code.

Monitoring is not something that we're likely to get around to during the experiment, so let's just have a dummy monitoring function that does nothing for now.

Reply:
Yep, I will do something simple, like just checking if the qemu pid is  still there. If not send fail. Otherwise send running.

Reply:
Sounds simple enough to implement, thumbs up to the idea.